src/main/java/io/cdap/plugin/gcp/bigquery/source/PartitionedBigQueryInputFormat.java

```diff
@@ -125,32 +125,9 @@ private String generateQuery(String partitionFromDate, String partitionToDate, S
     StringBuilder condition = new StringBuilder();
 
     if (timePartitioning != null) {
-      String columnName = timePartitioning.getField() != null ? timePartitioning.getField() : DEFAULT_COLUMN_NAME;
-
-      LegacySQLTypeName columnType = null;
-      if (!DEFAULT_COLUMN_NAME.equals(columnName)) {
-        columnType = tableDefinition.getSchema().getFields().get(columnName).getType();
-      }
-
-      if (partitionFromDate != null) {
-        if (LegacySQLTypeName.DATE.equals(columnType)) {
-          condition.append("TIMESTAMP(").append(columnName).append(")");
-        } else {
-          condition.append(columnName);
-        }
-        condition.append(" >= ").append("TIMESTAMP(\"").append(partitionFromDate).append("\")");
-      }
-      if (partitionFromDate != null && partitionToDate != null) {
-        condition.append(" and ");
-      }
-      if (partitionToDate != null) {
-        if (LegacySQLTypeName.DATE.equals(columnType)) {
-          condition.append("TIMESTAMP(").append(columnName).append(")");
-        } else {
-          condition.append(columnName);
-        }
-        condition.append(" < ").append("TIMESTAMP(\"").append(partitionToDate).append("\")");
-      }
+      String timePartitionCondition = generateTimePartitionCondition(tableDefinition, timePartitioning,
+              partitionFromDate, partitionToDate);
```

Comment:
style: when there is a line break in the middle of a method call, arguments should align with previous line

ex:
```
someMethod(arg1,
           arg2)
```